### ISSUE TITLE
Scrollview.goTo(x) goes bananas on iOS

### ISSUE THREAD

**Issue by guppyplopp** [dmvaldman/samsara#69]
Problem when jumping using .goTo when jumping backwards. 
Example: 
Jump to page 5 -> fine. 
Jump from 5 to page 1 -> The scrolling is superfast way beyond index 0. It bounces back and settles on page 0 (which was NOT what was wanted). 
Jumping again from this position to page 5 -> ends up at page 4. 
After this point almost anything can happen trying to jump. It can end with no slide at all, it can end up on any random slide, ... 

I have narrowed it down to it has something to do with the touching, not the OS. It works fine setting the .goTo(x) with code. But not when it's a 'touch' event starting it. 
The issue can be seen in Chrome (on MacOS at least) simulating iPad. 

Would be thankful for suggestions to fix this temporary since I have a deadline on Monday night...! 
In my project I'm not using free scrolling. I use it paginated for jumping between slides. So I guess I could remove the listening to touch scrolling, but I would like the touch clicking without the delay.

**Comment by frh3ddy** [comment]
I was not able to replicate it. if you are setting paginated to true, the the Scrollview size should be the same size as the individual slide, so if your slide is [200, 300], scrollview size should be also, [200, 300], if you can create a pen in codepen.io will be better.

**Comment by guppyplopp** [comment]
The size is of course [undefined, true]. The time when we could build elements in a website with fixed pixels is long gone. 
I removed touch from Generic input, and the crazy behavior stopped. But it is anyway an issue. 
Even worse is the other issue (#70) with the orientation change.

**Comment by frh3ddy** [comment]
are you setting [undefined, true] in a surface? or to the scrollview's size?

**Comment by guppyplopp** [comment]
I don't set anything to the Scrollview itself. The content is [undefined, true]. (sort of) The content is actually a lot of nested things. Views, Views containing Scrollviews, which contains sequential layouts and so on.

**Comment by frh3ddy** [comment]
did you try looking at the carousel example?

**Comment by guppyplopp** [comment]
Yes. But it hasn't the feature to jump directly from slide 5 to 1 (for example) so this issue can not be tested there. 
The resize issue can be seen in the Carousel demo too.
